src/System.Runtime.Serialization.Json/tests/DataContractJsonSerializer.cs

```diff
@@ -2393,6 +2393,95 @@ public static void DCJS_SquareWithDeserializationCallback()
         Assert.Equal(value.Area, actual.Area);
     }
 
+    [Fact]
+    public static void DifferentDateTimeStylesForDeserialization()
```

Comment:
Could you follow our name standard for the test and use the name starts with DCJS e.g. DCJS_DifferentDateTimeStylesForDeserialization?